src/components/FilterBar/FilterBar.stories.mdx

```diff
@@ -0,0 +1,134 @@
+    },
+    showSubmitAll: {
+      type: { name: "boolean", required: false },
+      defaultValue: false,
+      description: "`boolean`",
+    },
+    onSubmit: {
+      type: { name: "() => void", required: false },
+      description:
+        "Function to handle submit of all selectedItems - needs to be provided from consuming app",
+    },
+    onClear: {
+      type: { name: "() => void", required: false },
+      description:
+        "Function to clear all selectedItems - available through hook",
+    },
+    onOpen: {
+      contol: false,
+      description:
+        "Function to open the filter modal overlay on mobile viewport - available through hook",
+      type: { name: "() => void", required: false },
+    },
+    onClose: {
+      control: false,
+      description:
+        "Function to close the filter modal overlay on mobile viewport - available through hook",
+      type: { name: "() => void", required: false },
+    },
+    onToggle: {
+      control: false,
+      description:
+        "Function to toggle the filter modal overlay on mobile viewport - available through hook",
+      type: { name: "() => void", required: false },
+    },
+  }}
+/>
+
+# FilterBar
+
+| Component Version | DS Version |
+| ----------------- | ---------- |
+| Added             | `1.4.0`    |
+| Latest            | `1.4.0`    |
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Component Props](#component-props)
+- [Accessibility](#accessibility)
+- [Layout Patterns](#layout-patterns)
+- [useFilterBar hook](#usefilterbar-hook)
+
+## Overview
+
+<Description of={FilterBar} />
+
+## Component Props
+
+<Canvas withToolbar>
+  <Story
+    name="FilterBar"
+    args={{
+      id: "filterbar-id",
+      headingText: "FilterBar",
+      layout: "row",
+      showClearAll: "false",
+      showSubmitAll: "false",
+    }}
+  >
+    {(args) => <story.FilterBarStory {...args} />}
+  </Story>
+</Canvas>
+
+<ArgsTable story="FilterBar" />
+
+## Accessibility
+
+The `FilterBar` component is a wrapper component for components with filter functionality.
+The accessibility of `FilterBar` depends on the passed child components accessibility.
+See the accessibility details of valid FilterBar child components:
+
+- [MultiSelect](https://nypl.github.io/nypl-design-system/reservoir/v1/?path=/docs/components-form-elements-multiselect--multi-select-list-box#accessibility)
+- [MultiSelectGroup](https://nypl.github.io/nypl-design-system/reservoir/v1/?path=/docs/components-form-elements-multiselectgroup--multi-select-group#accessibility)
+
+## Layout Patterns
+
+The display of the FilterBar child components can be set to a "row" or a "column" layout.
+On screen width under 600px width the `layout` is automatically set to "column".
+Note: when `layout = "column"` is passed, the MultiSelect components will have `isBlockElement` set to `true` by default.
+
+<Canvas>
+  <DSProvider>
+    <story.FilterBarLayoutStory />
+  </DSProvider>
+</Canvas>
+
+## `useFilterBar` hook
```

Comment:
Add a to this section a direct link to the `useFilterBar` page under the `Hooks` heading.

Reply:
I moved the link that was at  the bottom of this section to the top. Pls let me know if I misunderstood the task.

Reply:
Oops.  I must have missed that initially.  I'm sorry.  Had the link not existed, the bottom of the section is where I would have suggested to put it.  However, I clearly misssed it when it was there, so leave it in the no position below the section heading.